storage/storage.go

```diff
@@ -392,18 +388,24 @@ func (storage DBStorage) ReadReportForCluster(
 	return report, types.Timestamp(lastChecked.UTC().Format(time.RFC3339)), err
 }
 
-// ReadRuleForReport reads rule result (health status) for selected cluster
-func (storage DBStorage) ReadRuleForReport(
+// ReadSingleRule reads rule result (health status) for selected cluster
+func (storage DBStorage) ReadSingleRule(
```

Comment:
Why this was changed?

Reply:
Same answer as in one of previous. I don't think "ForReport" makes any sense here, but again, it's too subjective, I can revert a strange version if you want.

Reply:
Basically here is used same logic as for cluster, we defined report and reading rule for this report. But ok, it could be `ReadSingleRule`